config/config.go

```diff
@@ -1369,9 +1371,20 @@ func getGoGCEnv() int {
 	return DefaultRuntimeConfig.GoGC
 }
 
+type otlpToPrometheusTranslationOption string
+
+var (
+	// KeepUTF8 will keep UTF-8 characters as they are, units and type suffixes will still be added.
+	KeepUTF8 otlpToPrometheusTranslationOption = "KeepUTF8"
+	// PrometheusRecommendations is the default option for translating OTLP to Prometheus.
+	// This option will translate all UTF-8 characters to underscores, while adding units and type suffixes.
+	PrometheusRecommendations otlpToPrometheusTranslationOption = "PrometheusRecommendations"
+)
+
 // OTLPConfig is the configuration for writing to the OTLP endpoint.
 type OTLPConfig struct {
-	PromoteResourceAttributes []string `yaml:"promote_resource_attributes,omitempty"`
+	PromoteResourceAttributes   []string                          `yaml:"promote_resource_attributes,omitempty"`
+	OTLPToPrometheusTranslation otlpToPrometheusTranslationOption `yaml:"otlp_to_prometheus_translation,omitempty"`
```

Comment:
This is already nested within an "otlp" config, and it is implied that it is translating to prometheus, so I think those should be omitted from the name.  I would prefer `name_and_key_translation`, `translation_strategy`, or something like that.